modules/videosegmenter-workflowoperation/src/main/java/org/opencastproject/workflow/handler/videosegmenter/VideoSegmenterWorkflowOperationHandler.java

```diff
@@ -110,6 +110,12 @@ public WorkflowOperationResult start(final WorkflowInstance workflowInstance, Jo
     }
     Track track = candidates.get(0);
 
+    // Skip operation if media is shorter than 15s since we won't generate a sensible segmentation on such a short
+    // video anyway.
+    if (track.getDuration() != null && track.getDuration() < 15000) {
```

Comment:
15 seconds is fine to me, although I somewhat agree with Matthias in that sub 5 minute videos don't really benefit from segmentation.

My complaint here whoever, is somewhat simpler: Magic numbers.  IMO this should be a constant properly defined so it shows up in the javadocs.